ee/models/event_definition.py

```diff
@@ -6,11 +6,11 @@
 
 class EnterpriseEventDefinition(EventDefinition):
     owner = models.ForeignKey("posthog.User", null=True, on_delete=models.SET_NULL, related_name="event_definitions",)
-    description: models.CharField = models.CharField(max_length=400, blank=True)
+    description: models.CharField = models.CharField(max_length=400, blank=True, null=True, default="")
     tags: ArrayField = ArrayField(models.CharField(max_length=32), null=True, blank=True, default=list)
     updated_at: models.DateTimeField = models.DateTimeField(auto_now=True)
     updated_by = models.ForeignKey("posthog.User", null=True, on_delete=models.SET_NULL, blank=True)
-    verified: models.BooleanField = models.BooleanField(default=False, null=False, blank=True)
+    verified: models.BooleanField = models.BooleanField(default=False, null=True, blank=True)
```

Comment:
I think the null=true here makes the `verified` field have 3 options: Null, False and True. We probably want to keep it to just the 2 options (False and True)